man/journald.conf.xml

```diff
@@ -358,6 +358,28 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>SyslogFacility=</varname></term>
+        <listitem><para>Sets the syslog facility to use when logging
+        to syslog. One of <option>kern</option>,
```

Comment:
I'm not sure I understand why this is useful. https://github.com/systemd/systemd/issues/5937 seems to be about preventing `journald` from storing/forwarding messages with `SYSLOG_FACILITY` which is greater than `MaxSyslogFacility`.